mqt/benchviewer/src/backend.py

```diff
@@ -150,14 +150,66 @@ def createDatabase(zip_file: ZipFile):
     return database
 
 
-def read_mqtbench_all_zip():
+def read_mqtbench_all_zip(
+    skip_question: bool = False,
+    target_location: str = "mqt/benchviewer/static/files/qasm_output",
+):
     global MQTBENCH_ALL_ZIP
-    huge_zip = Path("./static/files/qasm_output/MQTBench_all.zip")
+    huge_zip = Path(target_location + "/MQTBench_all.zip")
+    version_of_suitable_benchmark_zipfile = "v1.0.0"
+
+    url = (
+        "https://api.github.com/repos/nquetschlich/test/releases/tags/"
+        + version_of_suitable_benchmark_zipfile
+    )
+    response = requests.get(url)
+    if not response:
+        print("Suitable benchmarks cannot be downloaded, URL is faulty.")
+    else:
+        if not skip_question:
+            file_size = int((response.json()["assets"][0]["size"])) / 10e6
```

Comment:
is it guaranteed that the added `zip` file is asset number `0`? shouldn't there be a further sanity check that a proper zip has been selected? probably by checking the `name` against a suitable regex.

Furthermore, the division factor is wrong by a factor of 10 if I am not mistaken.
nitpick: shouldn't the division be by `2**20` instead of `1e6`?

Reply:
Good catches, I adapted the code such that it iterates now over all assets and checks for the correct name. 

Also adapted the division factor.